packages/browser/src/tracing/browserTracingIntegration.ts

```diff
@@ -288,7 +288,7 @@ export const browserTracingIntegration = ((_options: Partial<BrowserTracingOptio
           return;
         }
 
-        if (activeSpan) {
+        if (activeSpan && !spanToJSON(activeSpan).timestamp) {
```

Comment:
While debugging a test failure, I noticed that this surfaced another problem - we were potentially double-ending the span here. If the idle span already ended on it's own, we do not want to/need to end it anymore here! Calling this twice lead to some issues as it tried to remove the hooks twice, ...